### PR TITLE
fix(hstore): JRaft maxEntriesSize configuration parameters do not take effect

### LINKED ISSUE #2629: [Bug] JRaft maxEntriesSize configuration parameters do not take effect
### Bug Type (问题类型)

None

### Before submit

- [X] 我已经确认现有的 [Issues](https://github.com/apache/hugegraph/issues) 与 [FAQ](https://hugegraph.apache.org/docs/guides/faq/) 中没有相同 / 重复问题 (I have confirmed and searched that there are no similar problems in the historical issue and documents)

### Environment (环境信息)

- Server Version: master
- Backend: HStore

### Expected & Actual behavior (期望与实际表现)

application.yml 
```
raft:
  # raft 缓存队列大小
  disruptorBufferSize: 1024
  address: 127.0.0.1:8510
  max-log-file-size: 600000000000
  # 快照生成时间间隔，单位秒
  snapshotInterval: 1800
  maxEntriesSize: 2048

```

### Vertex/Edge example (问题点 / 边数据举例)

_No response_

### Schema [VertexLabel, EdgeLabel, IndexLabel] (元数据结构)

_No response_

### PR DESCRIPTION
**Pull request by JackyYangPassion**
## Purpose of the PR

- close #2629  

## Verifying these changes

- [ ] Trivial rework / code cleanup without any test coverage. (No Need)
- [ ] Already covered by existing tests, such as *(please modify tests here)*.
- [ ] Need tests and can be verified as follows:
    - xxx

## Does this PR potentially affect the following parts?

- [ ]  Dependencies ([add/update license](https://hugegraph.apache.org/docs/contribution-guidelines/contribute/#321-check-licenses) info & [regenerate_dependencies](../hugegraph-server/hugegraph-dist/scripts/dependency/regenerate_known_dependencies.sh)) 
- [ ]  Modify configurations
- [ ]  The public API
- [ ]  Other affects (typed here)
- [ ]  Nope

## Documentation Status

- [ ]  `Doc - TODO` 
- [ ]  `Doc - Done` 
- [ ]  `Doc - No Need`

### CODE REVIEW

File: hugegraph-store/hg-store-node/src/main/java/org/apache/hugegraph/store/node/grpc/HgStoreNodeService.java [hunk 1]
```diff
@@ -140,14 +141,14 @@ void addRaftTask(byte methodId, String graphName, Integer partitionId, Req req,
...
-            // 序列化，
+            // Serialization
             final byte[] buffer = new byte[req.getSerializedSize() + 1];
             final CodedOutputStream output = CodedOutputStream.newInstance(buffer);
             output.write(methodId);
             req.writeTo(output);
             output.checkNoSpaceLeft();
             output.flush();
-            // 传送给raft
+            //Add raft task
```
Comment (javeme) [inline hugegraph-store/hg-store-node/src/main/java/org/apache/hugegraph/store/node/grpc/HgStoreNodeService.java:151]: expect a space "// Add ..."

File: hugegraph-store/hg-store-node/src/main/java/org/apache/hugegraph/store/node/grpc/HgStoreNodeService.java [hunk 2]
```diff
@@ -190,7 +191,8 @@ public boolean invoke(int partId, byte[] request, RaftClosure response) throws
     }
 
     /**
-     * 处理raft传送过来的数据
+     * Process the data sent by raft
+     *
```
Comment (javeme) [inline hugegraph-store/hg-store-node/src/main/java/org/apache/hugegraph/store/node/grpc/HgStoreNodeService.java:195]: unused blank line

File: .asf.yaml [hunk 3]
```diff
@@ -28,7 +28,8 @@ github:
   del_branch_on_merge: true
   #labels:
   enabled_merge_buttons:
-    merge:  false
+     # TODO: disable it after common merged
+    merge:  true
```
Comment (javeme) [inline .asf.yaml:32]: seems we don't need to change in this commit

**Review by imbajin — APPROVED** [review]

File: .asf.yaml (hunk 3 above)
Comment (imbajin) [reply .asf.yaml:32]: > seems we don't need to change in this commit

Since another PR is blocked now

**Review by VGalaxies — APPROVED** [review]

**Pull request merged**